build/setup.d/19-install-ec2-instance-connect.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -ex
+
+VERSION="1.1.11"
+DOWNLOAD_URL="https://github.com/aws/aws-ec2-instance-connect-config/archive/${VERSION}.tar.gz"
+
+curl -L -o instance-connect.tar.gz "${DOWNLOAD_URL}"
+TEMP_DIR=$(mktemp -d)
+tar xzf instance-connect.tar.gz -C "$TEMP_DIR"
```

Comment:
It feels like checksum or signature verification is missing? @arjunrn

Reply:
I've made a [PR](https://github.com/zalando-stups/taupage/pull/577) to fix that. PTAL.